src/controllers/followController.js

```diff
@@ -0,0 +1,234 @@
+    try {
+      const foundUser = await User.find({
+        where: {
+          username
+        },
+      });
+      if (foundUser) {
+        const userId = request.userData.payload.id;
+        const user = await Follow.findAndCountAll({
+          where: {
+            followerId: userId
+          },
+          include: [
+            {
+              model: User,
+              as: 'following',
+              attributes: ['id', 'email', 'username']
+            }
+          ],
+          attributes: {
+            exclude: [
+              'id',
+              'userId',
+              'followerId',
+              'createdAt',
+              'updatedAt'
+            ],
+          }
+        });
+        if (user.rows.length === 0) {
+          return response.status(200).json({
+            message: 'You are not following any user'
+          });
+        }
+        const followingList = user.rows.map(follow => follow.following);
+        response.status(200).json({
+          followingCount: user.count,
+          following: followingList
+        });
+      } else {
+        return response.status(404).json({
+          message: 'User not found'
+        });
+      }
+    } catch (error) {
+      return response.status(500).json({
+        status: 'Fail',
+        error: error.message
+      });
+    }
+  }
+
+  /**
+     * @description Handles getting all users a user is following
+     * @param {object} request - Request object
+     * @param {object} response - Response object
+     * @returns {Array} An array of object containing required user data
+     * @memberof FollowController
+     */
+  static async getAllFollowers(request, response) {
+    const { username } = request.params;
+    try {
+      const foundUser = await User.find({
+        where: {
+          username
+        },
+      });
+      if (foundUser) {
+        const userId = request.userData.payload.id;
+        const user = await Follow.findAndCountAll({
+          where: {
+            userId
+          },
+          include: [
+            {
+              model: User,
+              as: 'myFollowers',
+              attributes: ['id', 'email', 'username']
+            }
+          ],
+          attributes: {
+            exclude: [
+              'id',
+              'userId',
+              'followerId',
+              'createdAt',
+              'updatedAt'
+            ],
+          }
+        });
+
+        if (user.rows.length === 0) {
+          return response.status(200).json({
+            message: 'You have no follower'
+          });
+        }
+        const followerList = user.rows.map(follow => follow.myFollowers);
+        response.status(200).json({
+          followerCount: user.count,
+          follower: followerList
+        });
+      } else {
+        return response.status(404).json({
+          message: 'User not found'
+        });
+      }
+    } catch (error) {
+      return response.status(500).json({
+        status: 'Fail',
```

Comment:
These error format should comply with [this](https://github.com/andela/balder-ah-backend#errors-and-status-codes)

Reply:
Thank you for spotting that, I will make corrections as soon as possible.